package.json

```diff
@@ -86,38 +85,39 @@
     "@types/react": "16.9.31",
     "@types/react-dom": "16.0.11",
     "babel-eslint": "10.0.1",
-    "babel-jest": "25.2.6",
+    "babel-jest": "24.9.0",
```

Comment:
Are you sure version of `babel-jest` is supposed to be downgraded?

Reply:
Good spot, I'll revert that, I ran `audit --fix` after going through the obvious ones not sure why that got downgraded

Reply:
https://github.com/Sage/carbon/commit/9e9c06ca20905858e4cd11e3e5d0b0db3280171d